diff --git a/drivers/input/touchscreen/fts_521/fts.c b/drivers/input/touchscreen/fts_521/fts.c
index 9c6aabb97d08..d5f5cbbe1c52 100644
--- a/drivers/input/touchscreen/fts_521/fts.c
+++ b/drivers/input/touchscreen/fts_521/fts.c
@@ -5012,7 +5012,7 @@ int fts_read_touchmode_data(void)
 	return ret;
 }
 
-static void fts_init_touchmode_data()
+static void fts_init_touchmode_data(void)
 {
 	int i;
 
@@ -5134,7 +5134,7 @@ static void fts_edge_rejection(bool on, int value)
 	return;
 }
 
-static void fts_update_grip_mode()
+static void fts_update_grip_mode(void)
 {
 	int i, ret;
 	u8 grip_cmd[9] = {0xc0, 0x08, 0x00,};
@@ -5255,7 +5255,7 @@ static void fts_update_grip_mode()
 	return;
 }
 
-static void fts_update_touchmode_data()
+static void fts_update_touchmode_data(void)
 {
 	bool update = false;
 	int i, j, ret = 0;
